detox/src/devices/AndroidDriver.test.js

```diff
@@ -0,0 +1,37 @@
+describe("AndroidDriver", () => {
```

Comment:
Drivers are not unit tested on purpose. The mostly touch command line tools and are just wrappers. Unit tests don't give much value. They are being tested in the e2e suite.

Reply:
This function is doing some regex voodoo so that's why I thought about adding one at least, but I'm ok with removing it.